src/main/java/org/jboss/processing/IssueCollector.java

```diff
@@ -2,39 +2,42 @@
 
 import com.atlassian.jira.rest.client.api.domain.Issue;
 import io.quarkus.logging.Log;
-import org.apache.camel.Exchange;
 import org.apache.camel.Processor;
 import org.apache.camel.component.jira.JiraEndpoint;
 import org.apache.camel.component.jira.consumer.NewIssuesConsumer;
+import org.jboss.jql.JqlBuilder;
 import org.jboss.processing.state.EveryIssueState;
 import org.jboss.processing.state.SingleIssueState;
+import org.jboss.query.IssueStatus;
+import org.jboss.query.SearchQuery;
 
 import java.util.List;
 import java.util.concurrent.CopyOnWriteArrayList;
 
-public class EveryIssueProcessor extends NewIssuesConsumer implements Executable<EveryIssueState> {
+public class IssueCollector extends NewIssuesConsumer implements Executable<EveryIssueState> {
 
     private static final State state = new State();
 
-    private EveryIssueProcessor(JiraEndpoint jiraEndpoint) {
-        super(jiraEndpoint, new Processor() {
-            @Override
-            public void process(Exchange exchange) {
-                Issue issue = exchange.getIn().getBody(Issue.class);
-                if (issue != null) {
-                    state.issueStates.add(new SingleIssueState(issue));
-                }
-            }
-        });
+    private static final Processor issueExtractor = exchange -> {
+        Issue issue = exchange.getIn().getBody(Issue.class);
+        if (issue != null) {
+            state.issueStates.add(new SingleIssueState(issue));
+        }
+    };
+
+    private IssueCollector(JiraEndpoint jiraEndpoint) {
+        super(jiraEndpoint, issueExtractor);
```

Comment:
it doesn't really make sense to have this separated in a field since it is only used here once.